src/scripts/underground/UndergroundUpgrade.ts

```diff
@@ -24,5 +24,9 @@ namespace UndergroundUpgrade {
         'Energy_Regen_Time',
         'Daily_Deals_Max',
         'Bomb_Efficiency',
+        'SurveyCharge_Cost',
+        'SurveyCharge_Efficiency',
+        'NewYLayer',
+        'Items_Min',
```

Comment:
We should shuffle these around so the `min items` upgrade is with the `max items` upgrade.

Reply:
Done. I have also shuffled the upgrades around so this is beneath Max Items also. I have also rebalanced the initial cost to be 500 in incremenents of 1500 instead of 1000. Same overall max-level price but more accessible early.